plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/MySqlQueryRunner.java

```diff
@@ -86,7 +86,7 @@ public static void main(String[] args)
             throws Exception
     {
         DistributedQueryRunner queryRunner = createMySqlQueryRunner(
-                new TestingMySqlServer(),
+                new TestingMySqlServer(false, true),
```

Comment:
I think this is one of the examples where we should consider using an environment variable to decide whether to bind to fixed or random port.

Multiple booleans side by side + 1 additional constructor (meant to be used only in *QueryRunner class) seems a little more work than required.

Reply:
Maybe it could be done as a follow up?